rust/cubenativeutils/src/wrappers/serializer/serializer.rs

```diff
@@ -138,7 +138,7 @@ impl<IT: InnerTypes> ser::Serializer for NativeSerdeSerializer<IT> {
     }
 
     fn serialize_none(self) -> Result<Self::Ok, Self::Error> {
-        Ok(self.context.undefined()?)
+        Ok(self.context.null()?)
```

Comment:
Are you sure `null` is correct? In JS it will result in smth like: `myVarOrProperty: null` instead of `undefined`.

Reply:
Yes, I think it's justified here. In general, this change was made to normally pass parameters in filters, which can be either a string or null.